test/System.Net.Http.Formatting.Test/System.Net.Http.Formatting.Test.csproj

```diff
@@ -32,7 +32,7 @@
     <Reference Include="System.Memory, Version=4.0.1.2, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
       <HintPath>..\..\packages\System.Memory.4.5.5\lib\netstandard2.0\System.Memory.dll</HintPath>
       <SpecificVersion>False</SpecificVersion>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
How does `Private=true` make this _more_ available? (Not blocking on this, just curious)

Reply:
It corresponds to `CopyLocal=true`, meaning the file will definitely appear in the test projects' bin/ folder.